runtime_lib/libxaie/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2023 AMD
+
+# cmake needs this line
+cmake_minimum_required(VERSION 3.1)
```

Comment:
We should update all of these to something sane, like cmake 20.1?

Reply:
agreed